libmscore/instrument.cpp

```diff
@@ -227,6 +229,12 @@ void Instrument::write(XmlWriter& xml, const Part* part) const
             xml.tag("transposeChromatic", _transpose.chromatic);
       if (!_instrumentId.isEmpty())
             xml.tag("instrumentId", _instrumentId);
+      switch (_staffGroup) {
+            case StaffGroup::PERCUSSION: xml.tag("staffGroup", 1); break;
+            case StaffGroup::TAB: xml.tag("staffGroup", 2); break;
+            }
```

Comment:
Although there are exceptions, we prefer words, e.g.:
```cpp
 case StaffGroup::PERCUSSION: xml.tag("staffGroup", "percussion"); break
 case StaffGroup::PERCUSSION: xml.tag("staffGroup", "tab"); break
```
For this, you could invest in a couple of helper functions like [Dynamic::speedToName](https://github.com/musescore/MuseScore/blob/b2dffcde4bb82a4f2a2c002c7ec77f64158ff6f2/libmscore/dynamic.cpp#L388) and Dynamic::nameToSpeed. Or you could do the quicker thing, which is hardcoding it. Your choice.

Reply:
OK, I will do that instead. I'll probably stick with hardcoding for now, as there would only need to be 2 options (with standard being the default), and it's unlikely that there'll ever be more.